### ISSUE TITLE
Is this repo dead?

### ISSUE THREAD

**Issue by cyraid** [RestComm/jdiameter#163]
Sorry to be that guy, but is this repository abandoned? There's pull requests but nobody merging them?

**Comment by matthias-p-nowak** [comment]
It seems so. There are spelling errors in naming. Some methods are outright misnomers, _createRequest_ returns a _Message_, _createUserDataRequest_ returns an _AppEvent_. An application needs to refer to the implementation part, hence a deployment in a container violates the implementation/api-separation principle. Some parts are only used by the test examples, but appear to be dead code inside the implementation. And it takes a lot of time to dive into the source code.

**Comment by cyraid** [comment]
Yeah. Kinda glad I made a C/C++ layer that interfaced with freediameter.
